Add notices section (#2841)
OffchainLabs/arbitrum-docs: docs/notices/fusaka-upgrade-notice.mdx

---
title: 'Fusaka-enabled Parent Upgrade Notice'
description: Upgrade notices for the transition to Fusaka
user_story: As an Arbitrum chain owner or node operator, I want to configure my chain/nodes to run on Fusaka-enabled parent.
content_type: notice
---

import { VanillaAdmonition } from '@site/src/components/VanillaAdmonition';

This upgrade notice explains what you need to do if you operate a chain or a node running on a Fusaka-enabled parent chain.

<VanillaAdmonition type="note">

> For ArbOS 50 release information, please visit our [Arbos release notes](run-arbitrum-node/arbos-releases/arbos50.mdx)

Reviewer: ```suggestion
For ArbOS 50 release information, please visit our [Arbos release notes](run-arbitrum-node/arbos-releases/arbos50.mdx)
```
```suggestion
For ArbOS 50 release information, please visit our [Arbos release notes](run-arbitrum-node/arbos-releases/arbos50.mdx)
```
```suggestion
For ArbOS 50 release information, please visit our [Arbos release notes](run-arbitrum-node/arbos-releases/arbos50.mdx)
```
Typo
```suggestion
For ArbOS 50 Dia release information, please visit our [ArbOS release notes](run-arbitrum-node/arbos-releases/arbos50.mdx)
```

> For ArbOS 50 release information, please visit our [Arbos release notes](run-arbitrum-node/arbos-releases/arbos50.mdx)

Author: "Go there" isn't a helpful link description.

</VanillaAdmonition>

## Date of required actions

> | Deadline            | Network upgrade                        | Who needs to upgrade                                                                       |

Reviewer: I dont have an issue with the use of tables, but arbos20, arbs32, and arbos40 all *do not have* tables and so i'm wondering if we have a strong case here to diverge from the precedent set prior?

| --------------------------------- | ------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------- |
| `October 14th 2025, 07:36:00 UTC` | Ethereum Sepolia Fusaka hard fork                                                          | Operators for Arbitrum chains settling on Ethereum Sepolia (Orbit L2s, Arbitrum Sepolia)  |
| `Dec 3rd 2025, 21:49:11 UTC`      | Ethereum Mainnet Fusaka hard fork                                                          | Operators for Arbitrum chains settling on Ethereum Mainnet (Orbit L2s, Arbitrum One/Nova) |
| `Dec 20th 2025, 17:00:00 UTC`     | Arbitrum Sepolia upgrade to [ArbOS 50](docs/run-arbitrum-node/arbos-releases/arbos50.mdx)  | Operators for Arbitrum Sepolia                                                            |
| N/A                               | Arbitrum One/Nova upgrade to [ArbOS 50](docs/run-arbitrum-node/arbos-releases/arbos50.mdx) | Operators for Arbitrum One/Nova                                                           |

> Arbitrum chain operators must ensure that their nodes are properly configured before or shortly after the parent chain upgrades to Fusaka or the child chain upgrades to ArbOS 50.

Author: Applied your requested change. Is that what you meant?

## For node operators

Outlined below are the configurations for each type of Arbitrum chain, along with node software required for operators.

> | Layer  | Data Availability    | Fallback to blobs   | Nitro node version                                                                                                                                                                                                                                           | Consenus Layer client                                                                                    |

Reviewer: Header fixes and typo fix
```suggestion
| Layer  | Data Availability    | Fallback to blobs   | Required Nitro node version                                                                                                                                                                                                                                           | Required configurations/setup for your Ethereum Consensus Client                                                                             |
```

| ------ | -------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------- |

> | **L2** | **Rollup**           | N/A                 | Upgrade to [`3.9.0`](https://github.com/OffchainLabs/nitro/releases/tag/v3.9.0) (ArbOS 50); [`3.8.0`](https://github.com/OffchainLabs/nitro/releases/tag/v3.8.0) or [`3.7.6`](https://github.com/OffchainLabs/nitro/releases/tag/v3.7.6) (ArbOS 40 or older) | Requires [historical blob data](#ensure-your-l1-ethereum-beacon-node-has-access-to-historical-blob-data) |

Reviewer: ```suggestion
| **L2** | **Rollup**           | N/A                 | Upgrade to [`3.9.0`](https://github.com/OffchainLabs/nitro/releases/tag/v3.9.0) (ArbOS 50); [`3.8.0`](https://github.com/OffchainLabs/nitro/releases/tag/v3.8.0) or [`3.7.6`](https://github.com/OffchainLabs/nitro/releases/tag/v3.7.6) (ArbOS 40 or older) | Needs the appropriate flags to serve all [historical blob data](#ensure-your-l1-ethereum-beacon-node-has-access-to-historical-blob-data) |
```

> | **L2** | **Rollup**           | N/A                 | Update to [`3.9.0`](https://github.com/OffchainLabs/nitro/releases/tag/v3.9.0) (ArbOS 50); [`3.8.0`](https://github.com/OffchainLabs/nitro/releases/tag/v3.8.0) or [`3.7.6`](https://github.com/OffchainLabs/nitro/releases/tag/v3.7.6) (ArbOS 40 or older) | Requires [historical blob data](#ensure-your-l1-ethereum-beacon-node-has-access-to-historical-blob-data) |

Reviewer: ```suggestion
| **L2** | **Rollup**           | N/A                 | Update to [`3.9.0`](https://github.com/OffchainLabs/nitro/releases/tag/v3.9.0) (ArbOS 50); [`3.8.0`](https://github.com/OffchainLabs/nitro/releases/tag/v3.8.0) or [`3.7.6`](https://github.com/OffchainLabs/nitro/releases/tag/v3.7.6) (ArbOS 40 or older) | Requires [all historical blob data](#ensure-your-l1-ethereum-beacon-node-has-access-to-historical-blob-data) |
```

| **L3** | **AnyTrust / AltDA** | Enabled or Disabled        | No nitro update required                                                                                                                                                                                                                                    | Doesn't require historical blob data                                                                                                |

### How to ensure your node has access to all historical blob data and blob data from all subnets

#### If you run a Nitro node and use an external L1 Ethereum beacon chain RPC:

Confirm that your external L1 beacon chain RPC provider has configured their L1 beacon chain node to subscribe to all subnets before the Ethereum Fusaka hard fork and have all historical blob data to ensure your Nitro node can sync from periods beyond the blob retention period (or that your RPC provider has backfilled the blob data properly).

#### If you run a Nitro node and operate your own L1 Ethereum beacon chain node:

Ensure that your consensus layer client & Nitro node is configured as per the table below.

> | Client        | Compatible with Nitro | Required Nitro Flag                                                 | Required flag for subscribing to all subnets  | Req Flag to serve historical blobs                             |

Reviewer: Fixed typo.
```suggestion
| Client        | Compatible with Nitro | Required Nitro Flag                                                 | Required flag for subscribing to all subnets  | Required Flag to serve historical blobs                             |
```

| ------------- | --------------------- | ------------------------------------------------------------------- | --------------------------------------------- | -------------------------------------------------------------- |
| Prysm         | ✅                    | None                                                                | `--subscribe-all-data-subnets`                | `--blob-retention-epochs`                                      |
| Lighthouse    | ✅                    | None                                                                | `--supernode`                                 | `--prune-blobs false` or `--blob-prune-margin-epochs <EPOCHS>` |

> | Teku          | ✅                    | `--parent-chain.blob-client.dangerous.skip-blob-proof-verification` | `--p2p-subscribe-all-custody-subnets-enabled` | N/A                                                            |

Reviewer: Remove the flag and fix syntax for the teku flag from https://docs.teku.consensys.io/reference/cli#p2p-subscribe-all-subnets-enabled
```suggestion
| Teku          | ✅                    | None | `p2p-subscribe-all-subnets-enabled` | N/A                                                            |
```

</VanillaAdmonition>

This document explains what you need to do if you operate a chain that settles on a Fusaka-enabled parent chain. This includes actions for Arbitrum Orbit chains as well as actions for Arbitrum One and Arbitrum Nova node operators.

## What's included in Fusaka

The Fusaka upgrade introduces several breaking changes. Review [EIP-7607](https://eips.ethereum.org/EIPS/eip-7607) for detailed information about the EIPs included in the Fusaka hard fork.

## Important dates

The following dates are relevant for Arbitrum chain operators.

| Date                              | Network upgrade                                                                            | Affected audience                                                                              |
| --------------------------------- | ------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------- |

> | `Dec 20th 2025, 17:00:00 UTC`     | Arbitrum Sepolia upgrade to [ArbOS 50](docs/run-arbitrum-node/arbos-releases/arbos50.mdx)  | Node operators for Arbitrum Sepolia                                                            |

Reviewer: Isn't this supposed to be Nov 20th?

> | N/A                               | Arbitrum One/Nova upgrade to [ArbOS 50](docs/run-arbitrum-node/arbos-releases/arbos50.mdx) | Node operators for Arbitrum One/Nova                                                           |

Reviewer: This is N/A because the vote hasn't passed yet right? Can we do this instead?
```suggestion
| Jan 8th, 2026, 17:00:00 UTC (proposed)                               | Arbitrum One/Nova upgrade to [ArbOS 50](docs/run-arbitrum-node/arbos-releases/arbos50.mdx) | Node operators for Arbitrum One/Nova                                                           |
```

> | `October 14th 2025, 07:36:00 UTC` | Ethereum Sepolia Fusaka hard fork                                                          | Node operators for Arbitrum chains settling on Ethereum Sepolia (Orbit L2s, Arbitrum Sepolia)  |

Reviewer: Fixing for consistency 
```suggestion
| `Oct 14th 2025, 07:36:00 UTC` | Ethereum Sepolia Fusaka hard fork                                                          | Node operators for Arbitrum chains settling on Ethereum Sepolia (Orbit L2s, Arbitrum Sepolia)  |
```

| **L3** | **Rollup**           | N/A                        | No nitro update required                                                                                                                                                                                                                                    | Doesn't require historical blob data                                                                 |
| **L3** | **AnyTrust / AltDA** | Enabled or Disabled        | No nitro update required                                                                                                                                                                                                                                    | Doesn't require historical blob data                                                                 |

> ### How to ensure your node has access to all historical blob data

Reviewer: ```suggestion
### How to ensure your node has access to all historical blob data and blob data from all subnets 
```
